### ISSUE TITLE
Font option for double-width characters.

### ISSUE THREAD

**Issue by fengyichui** [mintty/mintty#821]
Vim has two font options: `guifont`, `guifontwide`. The `guifontwide` is used for double-width characters.

I like `Consolas` to display English and `YaHei` for Chinese(double-width). `Consolas` display Chinese is ugly.

Could there be a option for double-width characters?

**Comment by mintty** [comment]
Duplicate of #580.
This is something that can be done, but there are some design issues:
* Should a secondary font be specific for CJK or should there be a more generic mechanism, so also a secondary font could be specified for other scripts? And how could such a mechanism be configured?
* Should Windows font fallback be considered in any way or should this simply override it?

**Comment by fengyichui** [comment]
Thanks very much! The fontlink worked!

The following dosbatch adds my favorite Chinese fonts:
```dosbatch
REG ADD "HKLM\Software\Microsoft\Windows NT\CurrentVersion\Fontlink\SystemLink" /v Consolas /t REG_MULTI_SZ /d "MSYH.TTC,Microsoft YaHei UI,128,96"\0"MSYH.TTC,Microsoft YaHei UI"\0"Arial.TTF,Arial,128,96"\0"Arial.TTF,Arial"
```

**Comment by mintty** [comment]
Mintty now supports script-specific secondary fonts, option `FontChoice`. Configuration example:
```
FontChoice=Hebrew:6;Arabic:7;Han:8;Hangul:9
Font6=David
Font7=Simplified Arabic Fixed
Font8=FangSong
Font9=MingLiU
```

**Comment by fengyichui** [comment]
👍👍👍

**Comment by mintty** [comment]
Released 3.0.2.
